src/SFA.DAS.ApplyService.Web/Views/Home/Error.cshtml

```diff
@@ -10,32 +8,9 @@
 <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            @if (HostingEnvironment.IsDevelopment())
```

Comment:
We removed this with agreement that in the startup we will configure to go to this page only if it is on hosted environment, we still need to see `DeveloperExceptionPage` when debugging. I have not found any config changes in this regards?